docs/chains/darwinia-smart-chain.md

```diff
@@ -37,6 +37,15 @@ smart contracts from the Ethereum chain to run their business. In this article w
     - `wss://pangoro-rpc.darwinia.network`
 
 
+### Tokens
```

Comment:
This section is a bit duplicated with the token info under https://docs.darwinia.network/chains/darwinia-smart-chain#network-info. Suggest removing the RING part and changing the title to `Other Tokens`